modules/misc/xfconf.nix

```
@@ -108,8 +108,11 @@ in {
         mkCommand = channel: property: value: ''
           $DRY_RUN_CMD ${pkgs.xfce.xfconf}/bin/xfconf-query \
             ${
-              escapeShellArgs
-              ([ "-n" "-c" channel "-p" "/${property}" ] ++ withType value)
+              if value != null then
+                escapeShellArgs
+                ([ "-n" "-c" channel "-p" "/${property}" ] ++ withType value)
+              else
+                escapeShellArgs ([ "-r" "-c" channel "-p" "/${property}" ])
```

Let's avoid repetition:

```suggestion
              escapeShellArgs
                ([ "-c" channel "-p" "/${property}" ] ++ if value == null then [ "-r" ] else [ "-n" ] ++ withType value)
```

Also, should we use `-R` with `-r`? (cf https://docs.xfce.org/xfce/xfconf/xfconf-query)